### DISCUSSION TITLE
I am not able to unlink an email associated with a Github account

### DISCUSSION THREAD

**Discussion by Jimit-Bavishi** [community/community#66383] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Before 2 years, I had created a Github account with an email. Unfortunately, I lost the recovery codes to that account. Now even though I know the password to this old account, I am not able to log into it.

I tried to follow the instructions in [Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account)

However, I never receive a verification email to unlink my account after step 8.

![Screenshot 2023-09-07 at 9 38 55 AM](https://github.com/community/community/assets/29938575/79cc4f32-7d6a-4c03-b674-19365437af9d)

I instead receive an email titled `[GitHub] Please review this attempt to sign in`. Github is thinking that my account is compromised.

<img width="1514" alt="Screenshot 2023-09-07 at 9 44 16 AM" src="https://github.com/community/community/assets/29938575/706b9695-96ea-4691-a9a8-4ce75116109d">

I am lost on how to unlink my email so I can associate it with another Github account. Could anyone please guide me?

**Comment by tridibbolar** [comment]
Hi,

I am facing the same issue with my corporate email ID. Long back, I had an account with that email, but I forgot the RECOVERY-CODE. My laptop also got changed. Now, whilst I am trying to unlink my account, I am not receiving any verification email but the email with "[GitHub] Please review this attempt to sign in" subject line. Can anyone help me to fix this issue?

Regards
Tridib
